internal/shared/db/model/setup.go

```diff
@@ -36,6 +36,8 @@ const (
 	V2FinalityProviderStatsCollection     = "v2_finality_providers_stats"
 	V2StakerStatsCollection               = "v2_staker_stats"
 	V2FinalityProvidersMetadataCollection = "v2_finality_providers_metadata"
+	// v3
+	BsnStatsCollection = "bsn_stats"
```

Comment:
I think we don't need to prefix collection with version related to phases, but comment above is sufficient